python/lsst/ts/scheduler/driver.py

```diff
@@ -92,15 +171,18 @@ def select_next_target(self):
         -------
         Target
         """
+        self.log.log(WORDY, 'Selecting next target.')
+        target = DriverTarget()
 
-        target = Target()
-
-        target.targetid = 0
+        self.targetid += 1
+        target.targetid = self.targetid
         target.num_exp = 2
         target.exp_times = [15.0, 15.0]
         target.num_props = 1
         target.propid_list = [0]
 
+        time.sleep(1.0)
```

Comment:
IMPORTANT: Please do not call `time.sleep` in an asyncio-related application. Instead make this a coroutine and call `await asyncio.sleep(1)`.

If this is being run in a thread then it probably doesn't matter. Otherwise...once you start using `asyncio` it tends to percolate down (much like `const` in C or C++).

Reply:
Yes. However, this method is not supposed to be a coroutine but rather a regular method that the CSC will run in a separate thread with `run_in_executor`. I added the regular `sleep` here to test that blocking it here won't block the execution of the event loop and stop the responsiveness of the CSC. 

I tested/validated the approach with larger values (like 30s). I was actually planning on taking it of for the final release but I was a bit torn about that.